packages/extension-ui/src/components/ErrorBoundary.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017-2020 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { WithTranslation } from 'react-i18next';
+
+import React from 'react';
+
+import translate from './translate';
+
+interface Props extends WithTranslation {
+  children: React.ReactNode;
+  doThrow?: boolean;
+  error?: Error | null;
+  onError?: () => void;
```

Comment:
I've a hard time figuring out how to make the best out of these, I'd remove them.

Reply:
I'll keep only what I think is useful, can bring it back if needed.